data/currencies.js

```diff
@@ -3,132 +3,198 @@ module.exports = {
+      monthly: ['25', '15', '10', '5']
+    }
   },
   'gbp': {
     code: 'gbp',
     minAmount: '1',
     symbol: '£',
-    presets: ['20', '10', '5', '3']
+    presets: {
+      single: ['25', '15', '10', '5'],
+      monthly: ['10', '7', '5', '3']
+    }
   },
   'hkd': {
     code: 'hkd',
     minAmount: '15',
     symbol: '$',
-    presets: ['80', '40', '20', '15']
+    presets: {
+      single: ['2000', '100', '500', '250'],
+      monthly: ['500', '300', '150', '75']
+    }
   },
   'huf': {
     code: 'huf',
     minAmount: '490',
     symbol: 'Ft',
-    presets: ['2800', '1400', '700', '500']
+    presets: {
+      single: ['5000', '3500', '2000', '1000'],
+      monthly: ['2000', '1500', '1350', '1000']
+    }
   },
   'ils': {
     code: 'ils',
     minAmount: '8',
     symbol: '₪',
-    presets: ['60', '30', '15', '10']
+    presets: {
+      single: ['500', '360', '180', '120'],
+      monthly: ['75', '40', '25', '15']
+    }
   },
   'jpy': {
     code: 'jpy',
     minAmount: '240',
     symbol: '¥',
-    presets: ['1200', '600', '300', '250']
+    presets: {
+      single: ['10000', '5000', '2500', '1000'],
+      monthly: ['5000', '2500', '1000', '500']
+    }
   },
   'mxn': {
     code: 'mxn',
     minAmount: '30',
     symbol: '$',
-    presets: ['160', '80', '40', '30']
+    presets: {
+      single: ['1000', '700', '500', '250'],
+      monthly: ['350', '200', '150', '90']
+    }
   },
   'nok': {
     code: 'nok',
     minAmount: '15',
     symbol: 'kr',
-    presets: ['80', '40', '20', '15']
+    presets: {
+      single: ['1500', '1000', '500', '250'],
+      monthly: ['400', '300', '250', '200']
+    }
   },
   'nzd': {
     code: 'nzd',
     minAmount: '3',
     symbol: '$',
-    presets: ['20', '10', '5', '3']
+    presets: {
+      single: ['150', '80', '50', '25'],
+      monthly: ['45', '40', '35', '30']
+    }
   },
   'php': {
     code: 'php',
     minAmount: '90',
     symbol: '₱',
-    presets: ['400', '200', '100', '90']
+    presets: {
+      single: ['5000', '3000', '2000', '1000'],
+      monthly: ['2000', '1000', '750', '500']
+    }
   },
   'pln': {
     code: 'pln',
     minAmount: '7',
     symbol: 'zł',
-    presets: ['40', '20', '10', '7']
+    presets: {
+      single: ['250', '100', '60', '40'],
+      monthly: ['90', '60', '45', '30']
+    }
   },
   'rub': {
     code: 'rub',
     minAmount: '100',
     symbol: 'руб',
-    presets: ['600', '300', '150', '100']
+    presets: {
+      single: ['1000', '500', '250', '150'],
+      monthly: ['250', '180', '120', '75']
+    }
   },
   'sek': {
     code: 'sek',
     minAmount: '15',
     symbol: 'kr',
-    presets: ['80', '40', '20', '15']
+    presets: {
+      single: ['1000', '500', '200', '100'],
+      monthly: ['250', '200', '150', '100']
+    }
   },
   'thb': {
     code: 'thb',
     minAmount: '70',
     symbol: '฿',
-    presets: ['400', '200', '100', '70']
+    presets: {
+      single: ['3000', '2000', '1000', '500'],
```

Comment:
UNICEF is a way bigger organization than us and their donors definitely not an average web users, so I don't think they will work for us here in my opinion.